strongbox-client/pom.xml

```diff
@@ -53,7 +61,80 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <argLine>-Xmx1G -XX:MaxPermSize=512m -XX:+UseConcMarkSweepGC -XX:+CMSClassUnloadingEnabled ${jacoco.agent.ut.arg}</argLine>
+                    <includes>
+                        <include>**/*.class</include>
+                    </includes>
+                </configuration>
+
+            </plugin>
+            <plugin>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <version>2.20.1</version>
```

Comment:
The plugin version should not be specified. This should be defined once in the `strongbox-parent` under the `<pluginManagement/>` section.